lib/tesla/middleware/telemetry.ex

```diff
@@ -24,7 +24,7 @@ defmodule Tesla.Middleware.Telemetry do
   @doc false
   def call(env, next, _opts) do
     {time, res} = :timer.tc(Tesla, :run, [env, next])
-    :telemetry.execute([:tesla, :request], time, %{result: res})
+    :telemetry.execute([:tesla, :request], %{request_time: time}, %{result: res})
```

Comment:
Does it work with both telemetry 0.3 and 0.4? If not, probably better to bump the dep from `mix.exs`.

Reply:
It works for both telemetry.